sklearn/preprocessing/_target_encoder.py

```diff
@@ -341,6 +342,3 @@ def _transform_X_ordinal(
         for f_idx, encoding in enumerate(encodings):
             X_out[indices, f_idx] = encoding[X_ordinal[indices, f_idx]]
             X_out[X_unknown_mask[:, f_idx], f_idx] = y_mean
-
-    def _more_tags(self):
-        return {"requires_y": True}
```

Comment:
Should this tag be removed? `TargetEncoder` does `require_y` in `fit` and `fit_transform`.